rust/exchange_rate/src/main.rs

```diff
@@ -203,12 +203,15 @@ async fn get_rate(job: Timestamp) {
     ic_cdk::api::print(url.clone());
 
     ic_cdk::api::print(format!("Making IC http_request call {} now.", job));
+
+    let bucket_start_time_index = 0;
+    let closing_price_index = 4;
     let request = CanisterHttpRequestArgument {
         url: url,
         method: HttpMethod::GET,
         body: None,
         max_response_bytes: Some(MAX_RESPONSE_BYTES),
-        transform: Some(TransformContext::new(transform, vec![])),
+        transform: Some(TransformContext::new(transform, vec![bucket_start_time_index, closing_price_index])),
```

Comment:
This works only because we use indexes that are fairly small numbers.

I am wondering if it won't be more explicit if we encode the indexes for example using "0 4".
And later we do something like

` let mut parts = s.split_whitespace().map(|s| s.parse::<i32>());
`